content/4.validators/2.becoming-a-validator/8.chain-upgrade.md

```diff
@@ -53,7 +51,7 @@ Navigate to the new directory and download the new release:
 ::highlight-card
 ```bash
 cd "${HOME}"/.archwayd/upgrades/0.x.y/bin
-wget -O archway.zip https://github.com/ArchwayNetwork/archway/releases/download/v0.x.y/archwayd-v0.x.y-linux-amd64.zip
+wget -O archway.zip https://github.com/archway-network/archway/releases/download/v1.0.0-rc.1/archwayd_v1.0.0-rc.1_linux_arm64.zip
```

Comment:
We should point to binaries directly instead of pointing to the zipped folders. e.g. 

linux/amd64: https://github.com/archway-network/archway/releases/download/v1.0.0-rc.1/archwayd_linux_amd64
linux/arm64: https://github.com/archway-network/archway/releases/download/v1.0.0-rc.1/archwayd_linux_arm64